modules/blocks.php

```diff
@@ -256,16 +255,20 @@ function jetpack_business_hours_render( $attributes, $content ) {
 	}
 
 	foreach ( $attributes['hours'] as $day => $hours ) {
-		$opening = strtotime( $hours['opening'] );
-		$closing = strtotime( $hours['closing'] );
-
-		$content .= '<dt class="' . esc_attr( $day ) . '">' .
-			ucfirst( $wp_locale->get_weekday( array_search( $day, $days ) ) ) .
-			'</dt>';
-		$content .= '<dd class="' . esc_attr( $day ) . '">';
-		if ( $hours['opening'] && $hours['closing'] ) {
-			$content .= sprintf(
-				/* Translators: Business opening hours info. */
+		$content    .= '<dt class="' . esc_attr( $day ) . '">' .
+					   ucfirst( $wp_locale->get_weekday( array_search( $day, $days ) ) ) .
+					   '</dt>';
+		$content    .= '<dd class="' . esc_attr( $day ) . '">';
+		$days_hours = '';
+
+		foreach ( $hours as $hour ) {
+			if ( ! $hour['opening'] && $hour['closing'] ) {
```

Comment:
This may not always be set, so I'd recommend checking with `empty` instead.

Did you mean to check for both values there instead?